### ISSUE TITLE
Better default for `pathed_env_vars`

### ISSUE THREAD

**Issue by MrLixm** [AcademySoftwareFoundation/rez#1569]
Hello,
After running into the issue, would that be possible to improve the `pathed_env_vars` configuration setting ?

Current state :

https://github.com/AcademySoftwareFoundation/rez/blob/371644d19de24d720cc0a4b14f6c17eeb61c55e1/src/rez/rezconfig.py#L558-L565

Suggested :

```python
pathed_env_vars = [
    "*PATH",
    "*PATHS",
]
```

I think it is safe to assume that all environment variables ending with `PATHS` can also be considered as "path like" and always need normalization.

Let me know if there is any side-effect that I did not think about.
I can take time to make a short PR for this change if you think that is a good idea.

Cheers.
Liam.

**Comment by JeanChristopheMorinPerso** [comment]
I think the safest we can do right now is `*PATH`, which is currently the default. The thing is that we have to be careful to not unnecessarily break anyone. Default values are there to cover some/most cases, not all cases.

**Comment by MrLixm** [comment]
Yes that is what makes me hesitant to post because that could be a breaking change.
Then what do you think of at least giving a hint in the documentation that it would be recommended to add this additional variable ?

**Comment by JeanChristopheMorinPerso** [comment]
https://github.com/AcademySoftwareFoundation/rez/pull/1475 will improve the documentation. You can see the improvements at https://rez--1475.org.readthedocs.build/en/1475/configuring_rez.html#pathed_env_vars.

If anything, we need user guides for our shells. Though, even with user guides, I wouldn't necessarily say "You should add x, y, y to x" and would more say "you might need to tweak these settings" to stay neutral.

**Comment by MrLixm** [comment]
thanks for sharing the pr, those are nice improvements.

> would more say "you might need to tweak these settings" to stay neutral.
yes totally
so perhaps this issue should be part of a bigger issue about "user guides" ? If so I am fine with closing this issue.

**Comment by JeanChristopheMorinPerso** [comment]
Yeah I think so. If you feel like it, you can comment on https://github.com/AcademySoftwareFoundation/rez/discussions/1277 to add docs for our shell plugins and link to your GH issue.

Additionally, we are looking for people to help us with docs, so if you want to contribute, adding docs would be a great way to contribute :)

**Comment by MrLixm** [comment]
Great, I try that.

**Comment by MrLixm** [comment]
Comment added to #1277.
Closing this.
